include/deal.II/lac/solver.h

```diff
@@ -36,33 +36,39 @@ template <typename number> class Vector;
  *
  * <h3>Requirements common to derived solver classes</h3>
  *
- * Since iterative solvers do not rely on any special structure of matrices or
- * the format of storage but only require that matrices and vectors define
+ * In general, iterative solvers do not rely on any special structure of matrices or
+ * the format of storage. Rather, they only require that matrices and vectors define
  * certain operations such as matrix-vector products, or scalar products
- * between vectors, this class as well as the derived classes and their member
+ * between vectors. Consequently, this class as well as the derived classes and their member
  * functions implementing concrete linear solvers are templated on the types
  * of matrices and vectors. However, there are some common requirements a
  * matrix or vector type must fulfill to qualify as an acceptable type for the
  * solvers in this hierarchy. These requirements are listed below.
  *
  * The classes we show below are not any concrete class. Rather, they are
- * intended to form a `signature' which a concrete class has to conform to.
+ * intended to form a "signature" which a concrete class has to conform to.
  * Note that the matrix and vector classes within this library of course
  * conform to this interface; therefore, SparseMatrix and Vector are good
  * examples for these classes as they provide the necessary signatures of
- * member functions.
- *
+ * member functions. (Although they also provide many more interfaces that
+ * solvers do not in fact need -- for example, element access.) In addition,
```

Comment:
This is no sentence on its own.

Reply:
Hm, not clear what you mean with this. The sentence seems clear to me!?

Reply:
The part in round brackets is a subordinate clause. It would make sense to join it with the previous sentence:
```
[...] they provide the necessary signatures of member functions (although [...]).
```
should be correct.

Reply:
Hm, sure. I can do that -- it leads to a rather long sentence, and I try to avoid this. But I don't feel too strongly about putting the final period before the parentheses or after :-)

New version now online.